### ISSUE TITLE
Indentation errors but why?

### ISSUE THREAD

**Issue by IschaGast** [brigade/scss-lint#462]
Yesterday I opened an issue but was forgotten that I set allow_non_nested_indentation to false. Now I changed  allow_non_nested_indentation to true but then I get an error.

This is in my settings file
``` 
  Indentation:
    enabled: true
    allow_non_nested_indentation: true
    character: space # or 'tab'
    width: 4

  SelectorFormat:
    enabled: true
    convention: hyphenated_BEM # hyphenated_lowercase # or 'strict_BEM', or 'hyphenated_BEM', or 'snake_case', or 'camel_case', or a regex pattern
```

But this code below still gives an indentation error but don't see why?
```
Ow now I know why I set the false value because when I set it to true I get this error, don't know how to solve this

>> scss-lint failed with error code: 70
>> and the following message:Error: Command failed: /bin/sh -c scss-lint -c
>> .scss-lint.yml static/scss/base/_buttons.scss static/scss/base/_fonts.scss
>> static/scss/base/_global.scss static/scss/base/_grid.scss
>> static/scss/base/_icons.scss static/scss/base/_print.scss
>> static/scss/components/_modules.scss static/scss/generic/_mixins.scss
>> static/scss/generic/_reset.scss static/scss/generic/_settings.scss
>> static/scss/generic/_utils.scss static/scss/ie.scss static/scss/style.scss
```
``` CSS
.grid {
    display: table;
    table-layout: fixed;
    width: 100%;
}
    .grid__unit {
        display: table-cell;
        padding-left: 10px;
        padding-right: 10px;
    }    
```

**Comment by sds** [comment]
Hey @IschaGast,

I can't reproduce the issue you're reporting using `scss-lint` 0.38.0. Can you run `scss-lint` directly on the problematic file and return the full stack trace? (it looks like you're running from another application or script)

**Comment by IschaGast** [comment]
This is what I get when I lint on my project folder:
```
SCSSLint::Linter::Indentation raised unexpected error linting file static/scss/generic/_settings.scss: 'undefined method `line' for nil:NilClass'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:194:in `node_indent'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:184:in `one_shift_greater_than_parent?'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:159:in `check_arbitrary_indent'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:53:in `check_indent_width'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:39:in `check_indentation'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:17:in `check_and_visit_children'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:61:in `visit_if'
/usr/local/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:131:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:62:in `visit_if'
/usr/local/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:131:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:142:in `block in visit_children'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:140:in `each'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:140:in `visit_children'
/usr/local/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:20:in `check_and_visit_children'
/usr/local/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:131:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:142:in `block in visit_children'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:140:in `each'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:140:in `visit_children'
/usr/local/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter/indentation.rb:10:in `visit_root'
/usr/local/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:131:in `visit'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/linter.rb:25:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/runner.rb:48:in `run_linter'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/runner.rb:29:in `block in find_lints'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/runner.rb:27:in `each'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/runner.rb:27:in `find_lints'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/runner.rb:17:in `block in run'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/runner.rb:16:in `each'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/runner.rb:16:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/cli.rb:54:in `scan_for_lints'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/cli.rb:48:in `act_on_options'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/lib/scss_lint/cli.rb:27:in `run'
/usr/local/lib/ruby/gems/2.2.0/gems/scss_lint-0.38.0/bin/scss-lint:6:in `<top (required)>'
/usr/local/bin/scss-lint:23:in `load'
/usr/local/bin/scss-lint:23:in `<main>'
```

**Comment by sds** [comment]
Thanks for the detailed stack trace, @IschaGast.

This looks like the issue that was reported in #446, which has been fixed.

I've released version [0.39.0](https://github.com/brigade/scss-lint/releases/tag/v0.39.0) with the fix.

**Comment by IschaGast** [comment]
Thanks, that fixed it :)
